index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ideefase PDF Generator</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/2.4.0/jspdf.umd.min.js"></script>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/2.2.19/tailwind.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<div class="container">
+    <form id="ideefaseForm" class="space-y-6" onsubmit="return validateForm();">
+    <!-- Header -->
+    <div class="header">
+        <header class="text-center">
+            <h1 class="title">Ideefase PDF Generator</h1>
+            <p class="subtitle">Genereer een overzichtelijk ideefase PDF</p>
+        </header>
+    </div>
+
+        <!-- Form Section -->
+    <div class="form-section">
+
+            <!-- Project Type -->
+            <div>
+                <label class="form-label">Project Type</label>
+                <p class="label-subtitle">Welke eindopdracht ga je maken?</p>
+                <div class="radio-group">
+                    <label class="radio-label">
+                        <input type="radio" name="projectType" value="Backend" checked>
+                        <span>Backend</span>
+                    </label>
+                    <label class="radio-label">
+                        <input type="radio" name="projectType" value="Frontend">
+                        <span>Frontend</span>
+                    </label>
+                    <label class="radio-label">
+                        <input type="radio" name="projectType" value="Fullstack">
+                        <span>Fullstack</span>
+                    </label>
+                </div>
+            </div>
+
+            <!-- Common Fields -->
+            <div>
+                <label for="projectName" class="form-label">Projectnaam</label>
+                <p class="label-subtitle">Hier mag je je applicatie een naam geven.</p>
+                <input type="text" id="projectName" class="form-input" placeholder="Bijv. E-commerce Website">
+            </div>
+            <div>
+                <label for="toelichting" class="form-label">Toelichting</label>
+                <p class="label-subtitle">Wat is het fictieve probleem dat je wil oplossen met jouw applicatie (het doel).</p>
+                <textarea  id="toelichting" class="form-textarea" rows="5" placeholder="Toelichting over het project"></textarea>
+            </div>
+            <div>
+                <label class="form-label">Kernfunctionaliteiten</label>
+                <div id="kernFunctionaliteitenContainer" class="space-y-2">
+                    <!-- Initial Fields -->
+                    <div class="functionaliteit-item">
```

Comment:
Ik snap dat je extra invoervelden (via JS) in een div'je moet wikkelen omdat er ook een verwijder-knop bij moet, maar voor de statische velden is het niet nodig ze allemaal in een div te zetten. Als je die inputs een `display: block` geeft komen ze ook onder elkaar :)